index.js

```diff
@@ -45,7 +44,7 @@ function fastifySwagger (fastify, opts, next) {
     } else {
       swaggerObject.info = {
         version: '1.0.0',
-        title: require(path.join(__dirname, 'package.json')).name || ''
+        title: ''
```

Comment:
can you use fs.readFileSync to read this?

Reply:
Yup, that's an alternative, I update